internal/node/npm_package_bin.bzl

```diff
@@ -26,8 +26,13 @@ def _expand_location(ctx, s):
         # We'll write into a newly created directory named after the rule
         outdir_segments.append(ctx.attr.name)
 
+    if not ctx.attr.output_dir:
+        if s.find("$@") != -1 and len(ctx.outputs.outs) > 1:
+            fail("$@ substitution may only be used with a single out")
```

Comment:
suggest to use `$(RULEDIR)` instead to make migration easier?

Reply:
Oh nevermind. Now I noticed the `if not ctx.attr.output_dir:`

Reply:
sure but there's a common case where you had only one out and won't get this message even though you need to change. May as well add it though, could help someone